hadoop-plugin-test/expectedOutput/negative/missingFields.out

```diff
@@ -45,6 +45,6 @@ RequiredFieldsChecker ERROR: TableauJob job19 must set workbookName
 RequiredFieldsChecker ERROR: HdfsWaitJob job20 must set dirPath, freshness, timeout, and forceJobToFail
 RequiredFieldsChecker ERROR: VenicePushJob job23 must set avroKeyField, avroValueField, clusterName, inputPath, veniceStoreName
 RequiredFieldsChecker ERROR: WormholePushJob job26 must set inputPath, namespace and dataset
-RequiredFieldsChecker ERROR: KabootarJob job27 must set trainingModelLocation, trainingName, wormholeNamespace, initialImport
+RequiredFieldsChecker ERROR: KabootarJob job27 must set trainingModelLocation, trainingName, aiProjectGroup, wormholeNamespace, initialImport. Please see http://go/kabootar-wiki for more details.
```

Comment:
There is no point in open sourcing the wiki since the plugin itself wont be open sourced. I have updated the error message not to show the wiki link.

Reply:
Technically the plugin is de-facto open source, since it is in a public repo on GitHub and is visible to anyone. Try accessing this page in Incognito mode to see for yourself.